spring-boot-data-dynamodb/src/test/java/com/baeldung/spring/data/es/repository/ProductInfoRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,84 @@
+package com.baeldung.spring.data.es.repository;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.baeldung.Application;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.IntegrationTest;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.amazonaws.services.dynamodbv2.model.ResourceInUseException;
+import com.baeldung.spring.data.es.model.ProductInfo;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = Application.class)
+@WebAppConfiguration
+@IntegrationTest
+@ActiveProfiles("local")
+@TestPropertySource(properties = { "amazon.dynamodb.endpoint=http://localhost:8000/", "amazon.aws.accesskey=test1", "amazon.aws.secretkey=test231" })
+public class ProductInfoRepositoryIntegrationTest {
+
+    private DynamoDBMapper dynamoDBMapper;
+
+    @Autowired
+    private DynamoDB dynamoDB;
+
+    @Autowired
+    private AmazonDynamoDB amazonDynamoDB;
+
+    @Autowired
+    ProductInfoRepository ProductInfoRepository;
+
+    private static final String EXPECTED_COST = "20";
+    private static final String EXPECTED_PRICE = "50";
+
+    @Before
+    @Ignore
+    public void setUp() throws Exception {
```

Comment:
You have ```@BeforeClass``` in the article, however you have ```@Before``` here. Please make it consistent. Based on the feedback, I think it should be ```@BeforeClass```. And the method needs to be ```static``` when we use ```@BeforeClass```